tests/csv2bufr/csv2bufr_test.py

```diff
@@ -1,18 +1,12 @@
 from eccodes import *
-from map2bufr import *
-
-from jsonschema import validate
-from io import StringIO, BytesIO
-import csv
-
+from csv2bufr import *
```

Comment:
As a rule, we should import explicitly.

Reply:
Complete, hopefully I've caught all the implicit imports.